### PR TITLE
Correctly hide hidden or expandUnder fields on review page

### PR DESCRIPTION
**Pull request by jbalboni**
Closes [1161](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1161)

This updates the review page to correctly show/hide conditionally hidden fields. 

screen shot 2017-02-15 at 3 00 43 pm
screen shot 2017-02-15 at 3 00 55 pm
screen shot 2017-02-15 at 3 01 06 pm

### CODE REVIEW

**Comment by cvalarida** [comment]
Looks good. :+1:

File: src/js/common/schemaform/helpers.js [hunk 1]
```diff
@@ -262,6 +266,43 @@ export function hasFieldsOtherThanArray(schema) {
 }
 
 /*
+ * Return a schema without array fields. If the schema has only array fields,
```
Comment (annekainicUSDS) [inline src/js/common/schemaform/helpers.js]: In what cases would the schema have only array fields?

**Review by annekainicUSDS — APPROVED** [review]
Looks good!

File: src/js/common/schemaform/helpers.js (hunk 1 above)
Comment (jbalboni) [inline src/js/common/schemaform/helpers.js]: It depends on the page design. The first iteration of the 1995 had a service periods page that was just the array field. Now it has a question, too, but it's still possible we'll encounter this in the future.

**Pull request merged**